### PR TITLE
Command line function to create file view

### PR DESCRIPTION
**Pull request by thomasyu888**
This will create a file view for users if they don't have one. No need for tests as this is tested in the python client.

### CODE REVIEW

File: synapsemonitor/__main__.py [hunk 1]
```diff
@@ -20,14 +20,27 @@ def monitor_cli(syn, args):
...
 
+def create_file_view_cli(syn, args):
+    """Create file view cli"""
+    fileview = monitor.create_file_view(
+        syn, name=args.name, project_id=args.project_id,
+        scope_ids=args.scope_ids
+    )
+    print("To monitor the files in your specified scope, "
+          "you can run the command line function:")
+    print(f"$ synapsemonitor view {fileview.id} --days 4")
```
Comment (mattfazza) [inline synapsemonitor/__main__.py:31]: Good use of f-strings, Tom

File: synapsemonitor/__main__.py [hunk 2]
```diff
@@ -70,6 +83,28 @@ def build_parser():
...
+    parser_create_view.add_argument(
+        'project_id',
+        help='Synapse Project Id to store file view in'
+    )
+    parser_create_view.add_argument(
+        '--scope_ids', nargs='+', required=True,
+        help='Synapse Folder / Project Ids'
+    )
+    parser_create_view.set_defaults(func=create_file_view_cli)
+
```
Comment (mattfazza) [inline synapsemonitor/__main__.py:107]: I'm surprised this doesn't already exist.  This is equivalent to an `ls` inside a scope.  Very cool

**Review by mattfazza — APPROVED** [review]
Couldn't catch anything, Tom!  Thanks for putting this together.  I can't believe this didn't exist before!

**Comment by thomasyu888** [comment]
Thanks @mattfazza for the review!

**Pull request merged**